lib/kitchen/provisioner/policyfile_zero.rb

```diff
@@ -112,7 +120,9 @@ def prepare_cookbooks
       #
       # @api private
       def policy_exporter
-        @policy_exporter ||= ChefDK::PolicyfileServices::ExportRepo.new(export_dir: sandbox_path)
+        # Must force this because TK by default copies the current cookbook to the sandbox
+        # See ChefDK::PolicyfileServices::ExportRepo#assert_export_dir_clean!
```

Comment:
Yeah, I was surprised to see this - by default any provisioner which extends `ChefBase` copies over the current cookbook in the sandbox dir.  Definitely something that should be changed in TK.